content/en/_index.html

```diff
@@ -73,7 +73,7 @@ <h5 class="panel-title">Secure</h5>
         </div>
 
         <div id="panel3" class="panel">
-            <a href="/docs/reference/config/policy-and-telemetry/">
+            <a href="/docs/tasks/policy-enforcement/enabling-policy/">
```

Comment:
@ericvn As per the current depreciation warning, 

> Mixer is deprecated. The functionality provided by Mixer is being moved into the Envoy proxies.

I can't find the appropriate link for the `Control` section of the home page. But as per [this page ](https://istio.io/docs/tasks/policy-enforcement/enabling-policy/), I can see the reference links in the warning section at the top. 

So I have added a link to [policy enforcement](https://istio.io/docs/tasks/policy-enforcement/enabling-policy/) page, but not sure if this is the right link.

Reply:
I think that the home page needs to be reworked, but for now I suggest pointing to the higher level config page, since the control functions are distributed into multiple different places now:
```suggestion
            <a href="/docs/reference/config/">
```

Reply:
Thanks @frankbu